src/cartridges/int_adyen_overlay/cartridge/scripts/adyenSessions.js

```diff
@@ -102,7 +102,7 @@ function createSession(basket, customer, countryCode) {
       sessionsRequest.shopperReference = customerID;
     }
 
-    sessionsRequest.blockedPaymentMethods = AdyenHelper.BLOCKED_PAYMENT_METHODS;
+    sessionsRequest.blockedPaymentMethods = AdyenHelper.getBlockedPaymentMethods();
```

Comment:
Should we include here the `blockedPaymentMethods.json` directly instead of using the `AdyenHelper`?